### ISSUE TITLE
Service errors

### ISSUE THREAD

**Issue by alanb128** [wolvi-lataniere/morpheus-serial#1]
After initial power up, my Pi did retrieve and display a new quote, and then powered off. However, the wake button does not seem to work and I'm seeing the following errors:
```
 inkyshot  ### Requesting Time Sleep
 inkyshot    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
 inkyshot                                   Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: morpheus-serial
 inkyshot  ERROR: Failed during request to http://morpheus-serial:5555/sleep_time/60/1800
Shutting down
Restarting service 'morpheus-serial sha256:f1216da778c79710563905294a3036f5baa375373d3e7b5edbfd73a381b7692f'
 morpheus-serial  standard_init_linux.go:219: exec user process caused: exec format error
Service exited 'morpheus-serial sha256:f1216da778c79710563905294a3036f5baa375373d3e7b5edbfd73a381b7692f'
```

**Comment by wolvi-lataniere** [comment]
@alanb128 Did the Pi really turned off, or was it a standard shutdown, with the Pi LEDs still ON.
From what I pick in the capture, `morpheus-serial` service doesn't seems to be running, this could come from a Pico without [firmware](https://github.com/wolvi-lataniere/morpheus-firmware/releases/download/v0.1.1/morpheus-serial-v0.1.1.uf2) flashed.

Regarding the 'wake button', by default the Inkyshot project uses TimeSleep (Only waking up after a set amount of time), you can change it to InputSleep to wake up on Pin activation replacing [this line](https://github.com/wolvi-lataniere/inkyshot-morpheus/blob/95c97af740951d218f84fdcdf08056afe36540d0/inkyshot/start.sh#L25) with `./morpheus.sh -a ${MORPHEUS_ADDR} -w InputSleep 0` (to wake up when GPIO3 is set low).

Note that in current state, the Morpheus Firmware only allows one wake up source at a time. It is possible to enable both with a few tweaks in the firmware and library though.

**Comment by alanb128** [comment]
Thanks for the info @wolvi-lataniere , yes I think you are correct, the initial Pi action was a shutdown.

**Comment by wolvi-lataniere** [comment]
@alanb128 I just noticed the `exec format error` message. I suppose you are running the 32bits version of the OS.
I checked with the balenaBlock and it was set to 64bits, so it will not be compatible. I've deleted it and re-created it as 32bits by default, adding a `morpheus-serial-aarch64` for 64 bits users (the 32bits should be compatible with both).
If you created your own fleet, try pushing a new release. If you joined the public fleet, just reboot your device, that should do the trick.
